packages/animations/lib/src/fade_transition.dart

```diff
@@ -66,22 +66,19 @@ class FadeTransitionConfiguration extends ModalConfiguration {
   /// a dismissible barrier. [barrierDismissible] cannot be null. If
   /// [barrierDismissible] is true, the [barrierLabel] cannot be null.
   FadeTransitionConfiguration({
+    Color barrierColor = Colors.black54,
```

Comment:
Does it make sense to make these configurable? I would just pass them directly in the constructor below:

```dart
super(
  barrierColor: Colors.black54,
  transitionDuration: const Duration(milliseconds: 150),
// etc.
)
```

Reply:
I was thinking maybe users might want to modify this transition for their own use. However, if we prefer that users are forced to use the values we provide them, we can just do this instead. Let me know what you think.

Reply:
We can also make them configurable. That's fine too.